development/views/index.html

```diff
@@ -13,6 +13,7 @@
       <div class="board collapsed" id="board-hell"></div>
       <div class="board collapsed" id="board-heaven"></div>
     </div>
+    <div class="inventories-container"></div>
```

Comment:
Since it holds a single inventory each time (or at least it should), you should probably rename it to `inventory-container`.